core/src/scss/utilities/mixins/link/_external-link.scss

```diff
@@ -13,17 +13,22 @@
 @mixin external-link($external-link: external-link, $external-link-hover: external-link-hover) {
   &::after {
     @include margin(null null -1px 4px);
-    background: url("#{$image-path}/#{$external-link}.png") no-repeat 0 0;
-    background: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
-    background-size: 100%;
-    content: '';
+    @include size(0.65em);
     display: inline-block;
-    height: 0.65em;
-    width: 0.65em;
+    content: '';
+    mask: url("#{$image-path}/#{$external-link}.svg") no-repeat 0 0;
+    mask-size: cover;
```

Comment:
This doesn't look well supported: https://developer.mozilla.org/en-US/docs/Web/CSS/mask-size#Browser_compatibility